frontend/src/general/SnackbarMessage.jsx

```diff
@@ -46,8 +46,8 @@ export default class SnackbarMessage extends React.Component {
             "aria-describedby": "message-id"
           }}
           anchorOrigin={{
-            vertical: "top",
-            horizontal: "right"
+            vertical: "bottom",
+            horizontal: "left"
```

Comment:
not really relevant to the PR but works fine! xD :rocket:

Reply:
it is relevant because with the menu, if you close a snackbar message, you might click logout instead. My moving it we remove the chance of accidentally logging out

Reply:
OH RIGHT LOL ty